connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/inbound/InboundConnectorReportingContext.java

```diff
@@ -45,4 +45,6 @@ public interface InboundConnectorReportingContext extends InboundConnectorContex
   Queue<Activity> getLogs();
 
   List<InboundConnectorElement> connectorElements();
+
+  Long getActivationTimestamp();
```

Comment:
Interesting mix of getters style 😄 but not related to you PR: `  List<InboundConnectorElement> connectorElements();`